src/main/java/org/broadinstitute/hellbender/cmdline/GATKPlugin/GATKReadFilterPluginDescriptor.java

```diff
@@ -366,7 +387,7 @@ public CountingReadFilter getMergedCountingReadFilter(final SAMFileHeader samHea
             clFilters.forEach(f -> finalFilters.add(f));
         }
 
-        return aggregateFunction.apply(finalFilters, samHeader);
+        return (finalFilters.isEmpty()) ? aggregateFunction.apply(null, samHeader) : aggregateFunction.apply(finalFilters, samHeader);
```

Comment:
Good catch! It is something that I changed for whatever reason in one of my multiple PRs and branches fixing the plugin and kept here because I didn't realize...

Simplified in next commit.